doc/asciidoc/consumer/configuration.adoc

```diff
@@ -68,21 +65,22 @@ streams.sink.topic.pattern.relationship.<TOPIC_NAME>=<RELATIONSHIP_EXTRACTION_PA
 streams.sink.enabled=<true/false, default=true>
 ----
 
-are still valid and they refer to Neo4j's default db instance, which is usually called `neo4j`, but can be controlled by separate Neo4j system configuration.
+are still valid and they refer to Neo4j's default db instance, which is usually called `neo4j`, but can be controlled by
+separate Neo4j system configuration.
 
-[abstract]
-.How manage the Neo4j's default db
---
- The default database is controlled by Neo4j's dbms.default_database configuration property so we're being clear about which default database applies for this user.
- Database names are case-sensitive, and must follow Neo4j database naming rules (Reference: https://neo4j.com/docs/operations-manual/current/manage-databases/configuration/#manage-databases-administration)
---
+[NOTE]
+====
+The default database is controlled by Neo4j's *dbms.default_database* configuration property so we're being clear about
+which default database applies for this user.
+Database names are case-insensitive and normalized to lowercase, and must follow Neo4j database naming rules.
+(Reference: https://neo4j.com/docs/operations-manual/current/manage-databases/configuration/#manage-databases-administration)
+====
 
-In particular the following two properties will be used ad default values
-for non-default db instances, in case of the specific configuration params are not provided:
+In particular the following property will be used as default values
+for non-default db instances, in case of the specific configuration params is not provided:
 
 ----
 streams.sink.enabled=<true/false, default=true>
-streams.procedures.enabled=<true/false, default=true>
```

Comment:
Why we removed this?

Reply:
Because it is a property related to Streams procedures and not directly related to Streams sink module neither to Sink multi-database support in particular. Furthermore, it was already specified in it's dedicated section (Neo4j Streams - Procedures). However i added a dedicated section for Streams procedures multi-database support.